contracts/starknet/teleport_oracle_auth.cairo

```diff
@@ -225,22 +320,22 @@ func request_mint{
 //     }
 func validate{
     syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, ecdsa_ptr: SignatureBuiltin*
-}(message: felt, sigs: Signature*, sigs_len: felt, threshold_: felt) {
+}(message: felt, signatures_len: felt, signatures: Signature*, threshold_: felt) {
     if (threshold_ == 0) {
         return ();
```

Comment:
shouldn't we revert here ?

Reply:
Validate will revert either when signature fails(line 336) or when there are less signatures than threshold(lines 324-328). 
Not sure if we should skip invalid signatures.

Reply:
I added logic to skip invalid signatures. Please rereview `request_mint/validate`.